<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTableWidget" name="tablaTurnos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>582</width>
     <height>498</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>582</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
width: auto;
border: 0;
}</string>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>110</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>37</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>35</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>23</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>NÂ° Turno</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fecha</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Medico</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Paciente</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Acciones</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="botonNuevoTurno">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>+ Turno</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFiltro">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Turnos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Paciente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medico</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="botonBuscar">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>5</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border: 0;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/loupe1.svg</normaloff>../icons/loupe1.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoBusqueda">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Busqueda...</string>
   </property>
  </widget>
  <zorder>botonBuscar</zorder>
  <zorder>tablaTurnos</zorder>
  <zorder>botonNuevoTurno</zorder>
  <zorder>comboBoxFiltro</zorder>
  <zorder>campoBusqueda</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
